plugins/amongus/Game.h

```diff
@@ -11,31 +11,41 @@
 #include "ProcessWindows.h"
 
 class Game {
-protected:
-	bool m_ok;
-	ptr_t m_client;
-	ptr_t m_playerControlStaticFields;
-	ProcessWindows m_proc;
-
 public:
-	inline bool isOk() const { return m_ok; }
+	enum Error { NONE, TEMP, PERM };
```

Comment:
In C++ code usually enum classes should be preferred unless you explicitly require the conversion to/from int frequently

Reply:
In that case I would move the enum outside `Game` and call it `GameError`. What do you think?

Reply:
Sounds good to me

Reply:
On another note though: Why do you even use a new enum for this? As far as I can tell you are only mirroring the states of the enum from the API, so why not use that directly?

Reply:
I just wanted to avoid including the header in `Game.h`.

In this case it actually doesn't make any difference.